src/components/index/sections/BetaPopUpDialog.tsx

```diff
@@ -0,0 +1,159 @@
+          maxHeight: '600px',
+          boxShadow: 'none',
+          overflow: {
+            xl: 'hidden',
+            lg: 'hidden',
+            md: 'hidden',
+            xs: 'auto',
+          },
+          margin: '0',
+        },
+      }}>
+      <Grid
+        sx={(theme) => ({
+          width: '600px',
+          margin: '0 auto',
+          padding: theme.spacing(7, 6, 0, 6),
+          border: '16px solid #E3E3E3',
+          borderRadius: '40px',
+          background: 'white',
+          [theme.breakpoints.down('md')]: {
+            width: '322px',
+            padding: theme.spacing(6, 1, 0, 1),
+            border: '8px solid #E3E3E3',
+          },
+        })}>
+        <DialogContent id="beta-pop-up-description" sx={() => ({ padding: 0 })}>
+          <BetaPopUpTextCard translationKey="index:beta-pop-up.beta-version" />
+          <BetaPopUpTextCard translationKey="index:beta-pop-up.feedback" />
+        </DialogContent>
+
+        <DialogActions
+          sx={() => ({
+            display: 'flex',
+            flexDirection: 'column',
+          })}>
+          <Button
+            onClick={handleClose}
+            sx={() => ({
+              margin: 'auto',
+              border: '2px solid #00B0FF',
+              borderRadius: '25px',
+              padding: '0 50px',
+              fontSize: '12px',
+              zIndex: 1,
+            })}>
+            {t('beta-pop-up.close')}
+          </Button>
+
+          <FormControl>
+            <FormControlLabel
+              control={
+                <Checkbox
+                  onChange={handleCheckboxChange}
+                  checkedIcon={<CheckCircleOutlineIcon />}
+                  icon={<CircleOutlined sx={() => ({ height: '24px' })} />}
+                />
+              }
+              label={
+                <Typography fontSize={12}>
+                  {t('beta-pop-up.do-not-show-again').toString()}
+                </Typography>
+              }
+            />
+          </FormControl>
+        </DialogActions>
+      </Grid>
+
+      <Grid
+        item
+        sx={(theme) => ({
+          display: 'flex',
+          justifyContent: 'space-between',
+          alignContent: 'flex-end',
+          position: 'relative',
+          margin: '0 auto',
+          top: '-125px',
+          width: '800px',
+          [theme.breakpoints.down('md')]: {
+            top: '-35px',
+            width: '370px',
+          },
+        })}>
+        <BetaManIcon
+          sx={(theme) => ({
+            height: '325px',
+            width: '180px',
+            [theme.breakpoints.down('md')]: {
+              height: '205px',
+              width: '115px',
+            },
+          })}
+        />
+        <BetaWomanIcon
+          sx={(theme) => ({
+            height: '325px',
+            width: '200px',
+            [theme.breakpoints.down('md')]: {
+              height: '205px',
+              width: '131px',
+            },
+          })}
+        />
```

Comment:
please remove the animated people holding the label. I understand they come from designers, but let's keep things simple and not overload the screen